espnet2/asr/transducer/beam_search_transducer_streaming.py

```diff
@@ -0,0 +1,1265 @@
+"""Search algorithms for Transducer models."""
```

Comment:
Hi @brianyan918 ! Not a review but I was wondering if you plan to adapt `beam_search_transducer.py` for streaming purposes instead? As far as I saw, the difference seem minors.

Also, what's the (future or current) purpose of `beam_search_default_v2`?